### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by chinmay7016**
fixes-#80
Added http://exifdata.com/ to image section #80

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -377,6 +377,9 @@ These can be useful for osint and social engineering.
 - [Sogou](https://pic.sogou.com/) - Chinese technology company that offers a search engine
 - [Jimpl](https://jimpl.com/) - Online photo metadata and EXIF data viewer
 - [Same Energy](https://same.energy/) - Find beautiful images
+- [Exid data](http://exifdata.com/)- look at your favorite images!
```
Comment (edoardottt) [inline README.md:380]: The description should be `online applicatation that lets you take a deeper look at your favorite images`

**Review by edoardottt — CHANGES_REQUESTED** [review]
requesting small changes

File: README.md [hunk 2]
```diff
@@ -377,6 +377,9 @@ These can be useful for osint and social engineering.
...
+
+
```
[the remark this answers is not in the document]
Comment (chinmay7016) [reply README.md]: Done sir as you requested pls review and merge it sir

File: README.md [hunk 3]
```diff
@@ -377,7 +377,7 @@ These can be useful for osint and social engineering.
 - [Sogou](https://pic.sogou.com/) - Chinese technology company that offers a search engine
 - [Jimpl](https://jimpl.com/) - Online photo metadata and EXIF data viewer
 - [Same Energy](https://same.energy/) - Find beautiful images
-
+- [Exid data](http://exifdata.com/)- online applicatation that lets you take a deeper look at your favorite images
```
Comment (edoardottt) [inline README.md:380]:
```
- [Exif data](http://exifdata.com/) - Online applicatation that lets you take a deeper look at your favorite images
```

plus a new line between this and threat intellingence below

**Review by edoardottt — CHANGES_REQUESTED** [review]
typos + no newline

**Pull request closed without merging**